Source/Parser/wp_parser.l

```diff
@@ -6,6 +6,7 @@
 %}
 
 %option nounput
+%option prefix="wxparser"
```

Comment:
@WeiqunZhang does the `wp_parser` prefix in the files here stand for `warp`(x)? In that case, I would also make the prefix `wpparser` instead.

Reply:
I think `wp` was chosen to stand for `W`arpX `P`arser.

Reply:
`wxparser` is fine.